packages/core/botpress-builtins/src/elements/builtin_action-button.js

```diff
@@ -59,9 +94,22 @@ export default {
     }
   },
 
-  uiSchema: {},
+  uiSchema: {
+    action: {
+      text: {
+        'ui:widget': 'textarea',
+        'ui:help': 'This text content will be sent via postback event',
+        'ui:options': {
+          rows: 3
+        }
+      },
+      phone_number: {
```

Comment:
camelcase

Reply:
This too comes from Facebook API, they don't use camelcase